### PR TITLE
Fix InlineDashboard test

### PR DESCRIPTION
**Pull request by samhere06**
Fix failing InlineDashboard test

### CODE REVIEW

File: packages/react-sdk-components/tests/e2e/Digv2/LandingPages/InlineDashboard.spec.js [hunk 1]
```diff
@@ -43,10 +47,12 @@ test.describe('E2E test', () => {
     const caseIdFilter = filters.locator('div:has-text("Case ID")');
     caseIdFilter.locator('input').fill(caseID);
 
-    await expect(page.locator(`td >> text=${caseID}`)).toBeVisible();
-    await expect(page.locator('td >> text="Complex  Fields" >> nth=1')).toBeVisible();
-    await expect(page.locator('td >> text="User DigV2"')).toBeVisible();
-    await expect(page.locator('td >> text="New" >> nth=1')).toBeVisible();
+    await responsePromise;
```
Comment (tumms2021389) [inline packages/react-sdk-components/tests/e2e/Digv2/LandingPages/InlineDashboard.spec.js:50]: change this to 
`await page.waitForResponse('**/data_views/D_ComplexFieldsList')`

**Review by tumms2021389 — CHANGES_REQUESTED** [review]

File: packages/react-sdk-components/tests/e2e/Digv2/LandingPages/InlineDashboard.spec.js (hunk 1 above)
Comment (samhere06) [reply packages/react-sdk-components/tests/e2e/Digv2/LandingPages/InlineDashboard.spec.js]: updated with change

File: packages/react-sdk-components/tests/e2e/Digv2/FormFields/OptionalAction.spec.js [hunk 2]
```diff
@@ -6,7 +6,9 @@ const common = require('../../../common');
 test.beforeEach(common.launchPortal);
 
 test.describe('E2E test', () => {
-  test('should login, create case and run the Email tests', async ({ page }) => {
+  let attributes;
```
Comment (tumms2021389) [inline packages/react-sdk-components/tests/e2e/Digv2/FormFields/OptionalAction.spec.js:9]: you can remove attributes variable from here

**Review by tumms2021389 — CHANGES_REQUESTED** [review]

File: packages/react-sdk-components/tests/e2e/Digv2/FormFields/OptionalAction.spec.js (hunk 2 above)
Comment (samhere06) [reply packages/react-sdk-components/tests/e2e/Digv2/FormFields/OptionalAction.spec.js]: updated with change

**Review by tumms2021389 — APPROVED** [review]

**Pull request merged**